src/main/app/models/draft.ts

```diff
@@ -1,3 +1,5 @@
 export class Draft {
   lastUpdateTimestamp: number
+  userEmail: string
+  userName: string
```

Comment:
little bit of a misnomer.
`idamName`?
`submitterName`?

Reply:
I would prefer to have it wrapped with some `owner` property. Then we have:

```
{
  ...
 owner: {
  email: "",
  name: ""
 }
}
```

or even more separate by having:

```
{
  ...
 metadata: {
  owner: {
   email: "",
   name: ""
  }
 }
}
```

By the way `lastUpdateTimestamp` will get removed from document when we switch to V3.